### DISCUSSION TITLE
Default parameters for Free Text annotation

### DISCUSSION THREAD

**Discussion by msn969** [sumatrapdfreader/sumatrapdf#2695] [Q&A]
Is there a way to change the default parameters of the Free Text annotation? Every time I add a new one, I'd have to change the following parameters: 
- Text Size: 8
- Text Color: Blue
- Border: 0

It'd be nice to be able to set these as default somewhere in the Advanced Settings document.

**Comment by kjk** [comment]
Not yet. I've added https://github.com/sumatrapdfreader/sumatrapdf/issues/2700 to track this.
